packages/@uppy/companion/src/server/controllers/s3.js

```diff
@@ -353,7 +357,62 @@ module.exports = function s3 (config) {
     }, next)
   }
 
+  const policy = {
```

Comment:
this IAM policy is hardcoded, will that be a problem for people self-hosting Companion who want to use their own policies?

Reply:
Probably no, but as often when it comes to AWS, I wouldn't be surprised to be wrong.

Reply:
is there a link to AWS docs which state which things are required to put in here? Would be nice to link that here.

Reply:
Good luck finding the relevant info, Artur and I came up with that through trial and error, with some help from OpenAI.